R/filter_confirmation.R

```diff
@@ -0,0 +1,473 @@
+#' Filter Confirmed Observations
+#'
+#' @description
+#'
+#' The function filters observation using a condition taking subsequent
+#' observations into account. For example, it could select all observations with
+#' `AVALC == "Y"` and `AVALC == "Y"` for at least one subsequent observation.
+#' The input dataset is joined with itself to enable conditions taking variables
+#' from both the current observation and the subsequent observations into
+#' account. The suffix ".join" is added to the variables from the subsequent
+#' observations.
+#'
```

Comment:
Add a non-Onco example of usage, e.g. "An example usage might be checking if a patient received two required medications within a certain timeframe of each other."

Reply:
Added